scripts/convert-to-f.js

```diff
@@ -46,11 +49,27 @@ function convertToFHandler() {
   debugger;
 
   // read user input
+  const celsius = prompt("Enter a celsius value: ");
+  let numberValue = Number(celsius);
+  
+  // before moving on ... type check!
+ 
+  if (isNaN(numberValue))
+   {alert ('please enter number');} 
+   
+  
 
   // core logic
-  const result = convertToF(c);
-
+  
+  const result =convertToF(celsius) ;
+  
   // display for use
+  var message=( celsius + '\xB0  is ' + result + ' \xB0 .');
```

Comment:
Could I pls propose to change the output a little: 
var message=( celsius + '\xB0 **C** is ' + result + ' \xB0 **F** .');

Reply:
that is good. let me try to change it.

Reply:
It is done.